guide/english/python/input-functions/index.md

```diff
@@ -58,3 +58,9 @@ As we know, the input are stored by default as a string, so we can use the map()
 NOTE: Inside the split() function, we can add the separator for the input, por example, if we want to separate each value by a comma, we should write input().split(",") and so on.
 
 <a href='https://docs.python.org/3/library/functions.html#input' target='_blank' rel='nofollow'>Official Docs</a>
+
+7\. Separating input of integers and store them in list:
+```
+l = list(map(int,input("Please provide input seperated by space : ").split()))
+```
+As we know, the input are stored by default as a string, so we can use the map() function, which basically map each value we get from the split with here in our case with int and convert them to int and at the same point of time each value that comes out of the map() get append to the list.
```

Comment:
I think we can split this sentence into 3 or 4 sentences. Otherwise, it might be confusing for the user. Perhaps we can change it to:

As we know, the input is stored by default as a string. So, we can use the map() function, which maps each value from the split() function (in our case, the string values will be mapped to integer values). After the mapping has taken place, the integer values will be appended to the list